Sources/Track.swift

```diff
@@ -22,7 +22,10 @@ class Track {
     weak var mixpanelInstance: MixpanelInstance?
     
     var isAutomaticEventEnabled: Bool {
```

Comment:
nit: this function can be removed for simplicity.

Reply:
removed